infrastructure/lambda/template.yaml

```diff
@@ -525,6 +525,7 @@ Resources:
           - src/get-addresses-handler.ts
         External:
           - "@aws-sdk/*"
+      provisionedConcurrency: 1
```

Comment:
This is defined wrong, see: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html#sam-function-provisionedconcurrencyconfig